### DISCUSSION TITLE
Welcome to IdentityDeveloperTemplates Discussions!

### DISCUSSION THREAD

**Discussion by Daniel-Krzyczkowski** [Daniel-Krzyczkowski/IdentityDeveloperTemplates#2] [General]
## 👋 Welcome!

 Do not hesitate to ask any questions you have about this project!

**Comment by staycoolkhan** [comment]
Hi Daniel,

Thanks for your great tutorials and blogs.

I'm following your tutorial from [TechMindFactory.com](http://techmindfactory.com/) and struggling to find this file "custom-policies-token-transformation-script.ps1" in your github. 

Please could you share this file to see what there inside.

Thanks in adavance.

Best Regards,

khalid
